### DISCUSSION TITLE
How do I Add Interaction Summaries for Cases?

### DISCUSSION THREAD

**Discussion by bethMorrisBroadacres** [deejay-hub/timeline-lwc#299] [Q&A] [answered]
Hi there,

I'm trying to add Interaction Summaries for the Case object, but it's not displaying against the Time Warp timeline. Could you please advise what I am doing wrong?

![image](https://github.com/user-attachments/assets/3dce0f2e-9baa-4069-8222-4a4b677e42e1)

Thanks

**Comment by deejay-hub** [comment]
The lookup relationship between Interaction Summaries and cases is called Deals.

<img width="1707" alt="image" src="https://github.com/user-attachments/assets/badde637-fbed-46a4-a1a1-cdaa335c6b4f">

You need to change the 2 fields highlighted as well as the Position Date Field. You don't need the prefix so just Start_Date_Time__c is fine. Just make sure the records you expect to see have a value populated. 

<img width="1302" alt="image" src="https://github.com/user-attachments/assets/4415f906-6e72-4bd9-9154-aa4ec8fbdde7">

**Reply by deejay-hub** [reply, marked as answer]
@bethMorrisBroadacres do you need any more help or did this put you on the right path?

**Reply by bethMorrisBroadacres** [reply]
That worked :) Thank you!

**Reply by bethMorrisBroadacres** [reply]
@deejay-hub could you please let me know where you got the list of relationship names from please?

**Reply by deejay-hub** [reply]
@bethMorrisBroadacres the list is dictated by what the developer or admin called the relationship name. For you, above, you can see that the field with the lookup or master/detail relationship has this value.
https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_relationships_and_custom_objects.htm

If you want to see all relationships for every object then you can print the Enterprise WSDL here under Setup -> Integrations -> API. Then click the Generate Enterprise WSDL link.
<img width="1721" alt="image" src="https://github.com/user-attachments/assets/4f48d8be-f576-46ed-9c4a-300fca00cd16">

This then prints a list of every object, field and relationship. You can search for the parent object then look for any record labelled 'QueryResult' to see the relationship name.

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/49539675-89d0-44d1-b22f-008b9992b048">

**Comment by deejay-hub** [comment]
You can also use the standard colour and icon if needed

<img width="1175" alt="image" src="https://github.com/user-attachments/assets/3a8e32e5-906f-4546-9958-f2971d380d92">
